deploy.sh

```diff
@@ -0,0 +1,14 @@
+# install and deploy backend
+git pull origin master
+npm i
+npm audit fix --force
+
+echo "\n\nInstalled Backend Requirements.. generating artifacts"
+npx prisma generate
```

Comment:
are we sure we want to generate, on an update?  if there is a database migration we should probably stop and do this manually